addons/account/views/mail_message_views.xml

```diff
@@ -63,12 +64,9 @@
         <field name="view_mode">list</field>
         <field name="help" type="html">
             <p class="o_view_nocontent_smiling_face">This is the Audit Trail Report</p>
-            <div>Make sure you first activate the audit trail in the accounting settings</div>
-            <a class="btn-link" type="action" name="%(action_account_config)d">Go to the accounting settings</a>
         </field>
         <field name="domain">[
             ('message_type', '=', 'notification'),
-            ('account_audit_log_activated', '=', True),
```

Comment:
We should only see the models that we want to track for accounting (i.e. see the `DOMAINS` keys)